packages/eslint-plugin/src/rules/naming-convention-utils/enums.ts

```diff
@@ -46,6 +46,9 @@ enum Selectors {
 
   // other
   import = 1 << 18,
+
+  // auto accessor
+  autoAccessor = 1 << 19,
```

Comment:
[Style] Nit - this feels like it belongs in `memberLike`. It's used with classes. Thoughts?

Reply:
I think it just might be a breaking change.

I mean, for those people who are using `memberLike` grouped accessor and arent following the naming conv for auto-accessor.

But, I dont really know, I'm very new to this btw.

Reply:
As per [our versioning guidelines](https://typescript-eslint.io/users/versioning#eslint-plugin-and-eslint-plugin-tslint)

> A change to the plugins shall not be considered breaking if it:
> ...
> - Adds additional checks to an existing rule that causes new reports in a small-to-medium set of cases in an average codebase.
> ...

We do not classify this as a breaking change as in most codebases there shouldn't be too many violations because (a) auto-accessor is very new and (b) it's likely that the code already follows the naming convention for other members